stable/app/values.yaml

```diff
@@ -0,0 +1,88 @@
+# Default values for app.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
```

Comment:
Is it possible to skip this? SInce this is an app chart, it should not ideally have any image.

Reply:
it was generated from `helm create <chart-name>` command, we were keeping this because it is the default value from there and also to make this chart alone is installable/deployable.
cc @rohilsurana

Reply:
That's the main point, it's the `app` chart. So for intent, it should be able to install any app that is specified. 
If we set something as default then it is that default app's(in this case nginx) chart.
I would suggest we remove the default value so that chart fails without specifying this valaue.